server/services/interventionsService.ts

```diff
@@ -63,7 +63,7 @@ export default class InterventionsService {
   }
 
   async createDraftReferral(token: string): Promise<DraftReferral> {
-    const restClient = await this.createRestClient(token)
+    const restClient = this.createRestClient(token)
```

Comment:
Nice one getting rid of these, I'd added a section to the tech debt checklist (or whatever it's called) for this, as it's been on my mind for a while.

Not to worry about for now, but in future, it might be a bit clearer to make changes like this in their own commit, just in case someone coming back to the project sees they were removed in this commit (about adding a list of draft referrals), when it's not _really_ related.

Reply:
agree - not seperating behaviour changes from refactoring is something i'm often guilty of and trying to change.